packages/wordcount/src/stripConnectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Replaces items matched in the regex with spaces.
  *
- * @param {Object} settings The main settings object containing regular expressions
+ * @param {import("./index").WordCountSettings} settings The main settings object containing regular expressions
```

Comment:
I wish it could be automatically enforced via one of either ESLint (e.g. the JSDoc plugin in use) or by Prettier, but I'd think we should standardize quotes usage to use the same single-quote convention otherwise standard:

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#strings

Reply:
Oh, that's my bad. Fixed in 244b8a131f. And agreed that it would’ve been nice for the tools to have taken care or warned me earlier.